OpenRA.Server/Program.cs

```diff
@@ -83,6 +83,7 @@ static void Run(string[] args)
 			{
 				// HACK: The engine code *still* assumes that Game.ModData is set
 				var modData = Game.ModData = new ModData(mods[modID], mods);
+				modData.MapCache.LoadPreviewImages = false;
```

Comment:
It might be worth adding a PERF comment.

Reply:
Added